src/components/common/EntityInfo.vue

```diff
@@ -73,7 +73,7 @@ import { CorpTypeCd, FilingNames, FilingTypes } from '@/enums'
 import { ContactPointIF, RegistrationStateIF } from '@/interfaces'
 import { EnumMixin, DateMixin } from '@/mixins'
 import { StaffComments } from '@bcrs-shared-components/staff-comments'
-import { axios } from '@/utils'
+import { axiosInstance as axios } from '@/utils'
```

Comment:
Note: This is DIFFERENT from `import axios from 'axios'`. This is our local instance of axios (that we modified) whereas the other one is the unmodified library itself.

This matters because, if we use our instance of axios, then we get a CORS error when running the code locally and accessing some APIs (eg, the webchat API).

Here, we want to use our modified instance.